hoomd/md/pair/pair.py

```diff
@@ -10,6 +10,7 @@
 from hoomd.md.nlist import NList
 from hoomd.data.parameterdicts import ParameterDict, TypeParameterDict
 from hoomd.data.typeparam import TypeParameter
+import numpy
```

Comment:
I realize I'm against the bandwagon here, but just because something is recommended in the docs doesn't mean it is a good practice. In this case the proliferation of `import numpy as np` provides a fluid representation that serves no purpose other than to make example code look cute. In contrast with what the linked documentation says, fluid representations make concepts more difficult to understand, not easier. If the authors truly intend for all users to use `np`, then they should have named the library `np`.

I personally find it more useful to `import numpy` and  copy and past method names and arguments that use the full name from the documentation (e.g. `numpy.testing.assert_allclose` documented here: https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html). The full names appear in the documentation more often than the short versions. As such, HOOMD has a mix of using numpy as `numpy` and `np` in different files.